packages/rum-core/src/browser/htmlDomUtils.spec.ts

```diff
@@ -61,10 +61,16 @@ if (!isIE()) {
   describe('isShadowRoot', () => {
     const parent = document.createElement('div')
     parent.attachShadow({ mode: 'open' })
+    const form = document.createElement('form')
+    const input = document.createElement('input')
+    input.setAttribute('name', 'host')
+    form.appendChild(input)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it took me a bit of time to understand what was going on here, it may be interesting to extract a dedicated test for it:
- we could clarify this case in the test name
- it would isolate this test setup from the simpler ones